hopsworks-common/src/main/java/io/hops/hopsworks/common/dao/featurestore/feature/FeaturestoreFeature.java

```diff
@@ -113,11 +113,11 @@ public void setType(String type) {
     this.type = type;
   }
 
-  public int getPrimary() {
+  public boolean getPrimary() {
```

Comment:
@SirOibaf I think it's better to not use primitives win DAOs. We try to follow this convention, albeit with not 100% success

Reply:
Ok fixed, in any case the schema does not accept nulls and set by default 0